src/App.js

```diff
@@ -1,36 +1,25 @@
 import {
-  BrowserRouter as Router, Routes, Route, Outlet,
+  BrowserRouter as Router, Routes, Route,
 } from 'react-router-dom';
-import { Provider } from 'react-redux';
-import store from './Redux/store';
 import './App.css';
+import { Provider } from 'react-redux';
 import SideBar from './components/sidebar/SideBar';
 import HousesDetails from './components/houses details/HouseDetails';
 import Houses from './components/houses/Houses';
-import Login from './components/auth/Login';
-import Register from './components/auth/Register';
-import AddHouse from './components/add hosue/addHouse';
-
-const SidebarLayout = () => (
-  <>
-    <SideBar />
-    <Outlet />
-  </>
-);
+import Reservations from './components/Reservations/Reservations';
+import Booking from './components/booking/Booking';
 
 function App() {
   return (
     <div className="App">
-      <Provider store={store}>
+      <Provider>
```

Comment:
You did not provide any store to the provider.

Reply:
I'm working on that now. Thanks

Reply:
Fixed it